Modules/_testcapi/unicode.c

```diff
@@ -220,6 +220,112 @@ unicode_copycharacters(PyObject *self, PyObject *args)
     return Py_BuildValue("(Nn)", to_copy, copied);
 }
 
+/* Test PyUnicode_ToLower() */
+static PyObject *
+unicode_tolower(PyObject *self, PyObject *arg)
+{
```

Comment:
It seems like this code is duplicated 4 times. Would it be possible to write a helper function to factorize the code? And pass PyUnicode_ToLower function pointer to it.